aptos-move/move-examples/parallel_accounts/sources/parallel_accounts_demo.move

```diff
@@ -0,0 +1,129 @@
+/// A demo of how multiple accounts can be used to delegate to a single resource
+/// account for parallelism between transactions.
+module parallel_accounts::parallel_accounts_demo {
+    use aptos_framework::aptos_account;
+    use aptos_token::token_transfers::offer_script;
+    use std::string::String;
+    use aptos_framework::account::{SignerCapability, create_resource_account, create_signer_with_capability, create_resource_address};
+    use std::vector;
+    use std::signer::address_of;
+
+    /// No transfer admin capability found
+    const ENO_ADMIN_CAP_FOUND: u64 = 1;
+    /// No delegated transfer capability found
+    const ENO_TRANSFER_CAP_FOUND: u64 = 2;
+    /// No transfer resource found
+    const ENO_TRANSFER_RESOURCE_FOUND: u64 = 3;
+
+    const DEMO_SEED: vector<u8> = b"parallel_demo";
+
+    /// Resource on the single transfer account
+    struct TransferResource has key {
+        signer_cap: SignerCapability,
+    }
+
+    /// Transfer admin resource, allows for delegation to accounts
+    struct TransferAdmin has key {
+        admin_cap: AdminCapability,
+    }
+
+    /// Transfer delegation account, which can only do transfers
+    struct TransferDelegate has key, drop {
+        transfer_cap: TransferCapability
+    }
+
+    /// Admin capability to delegate transfer actions
+    struct AdminCapability has drop, store { account: address }
+
+    /// Allows owner to do transfer actions
+    struct TransferCapability has drop, store { account: address }
+
+    /// Creates a resource account and stores the capability
+    public entry fun create_resource_account_and_store_cap(sender: &signer, seed: vector<u8>) {
+        let (resource_signer, signer_cap) = create_demo_resource_account(sender, seed);
+
+        // Store
+        move_to(&resource_signer, TransferResource {
+            signer_cap
+        });
+
+        move_to(sender, TransferAdmin {
+            admin_cap: AdminCapability {
+                account: address_of(&resource_signer)
+            },
+        });
+    }
+
+    /// Adds a transfer cap to an account, transaction must be multi-agent
+    public entry fun delegate_transfer_cap(admin: &signer, delegate: &signer) acquires TransferAdmin {
```

Comment:
likely better to set up some form of ACL for distributing these ... probably something that is kinda missing from our standard library... A trivial vector<address> of unclaimed would do....